arcgis-ios-sdk-samples/Layers/Blend renderer/BlendRenderer.storyboard

```diff
@@ -29,223 +29,185 @@
-                            <constraint firstItem="Np5-Md-wF5" firstAttribute="leading" secondItem="3Ub-iF-hKv" secondAttribute="leading" id="a6w-Lg-byB"/>
                             <constraint firstItem="74L-H9-OmC" firstAttribute="bottom" secondItem="FsE-cl-COY" secondAttribute="bottom" id="eim-Ao-5qY"/>
                             <constraint firstItem="FsE-cl-COY" firstAttribute="leading" secondItem="74L-H9-OmC" secondAttribute="leading" id="fG5-Vp-pgu"/>
                             <constraint firstItem="55E-Cz-Pew" firstAttribute="top" secondItem="3Ub-iF-hKv" secondAttribute="top" id="gtt-80-zIA"/>
                             <constraint firstItem="FsE-cl-COY" firstAttribute="trailing" secondItem="74L-H9-OmC" secondAttribute="trailing" id="hfi-Mf-O6c"/>
                             <constraint firstAttribute="trailing" secondItem="55E-Cz-Pew" secondAttribute="trailing" id="hmE-1O-4Fg"/>
-                            <constraint firstAttribute="bottom" secondItem="Np5-Md-wF5" secondAttribute="bottom" id="kzf-3S-7M1"/>
                             <constraint firstItem="55E-Cz-Pew" firstAttribute="top" secondItem="74L-H9-OmC" secondAttribute="top" id="q61-y2-FM3"/>
                         </constraints>
                         <viewLayoutGuide key="safeArea" id="74L-H9-OmC"/>
                         <variation key="default">
                             <mask key="constraints">
                                 <exclude reference="gtt-80-zIA"/>
-                                <exclude reference="Qls-Mi-NY3"/>
                             </mask>
                         </variation>
                     </view>
                     <connections>
                         <outlet property="mapView" destination="55E-Cz-Pew" id="8jK-Q9-3f1"/>
-                        <outlet property="visualEffectView" destination="Np5-Md-wF5" id="Rcq-Gr-0gP"/>
                     </connections>
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="Wf4-dx-wNO" sceneMemberID="firstResponder"/>
             </objects>
             <point key="canvasLocation" x="305" y="372"/>
         </scene>
         <!--Blend Renderer SettingsVC-->
-        <scene sceneID="8lg-xS-cBk">
+        <scene sceneID="zuR-Aq-sif">
             <objects>
-                <viewController id="mbD-bO-mFc" customClass="BlendRendererSettingsVC" customModule="arcgis_ios_sdk_samples" customModuleProvider="target" sceneMemberID="viewController">
-                    <view key="view" contentMode="scaleToFill" id="PAD-ae-PsN">
-                        <rect key="frame" x="0.0" y="0.0" width="300" height="220"/>
+                <tableViewController id="VDk-RB-4PB" customClass="BlendRendererSettingsVC" customModule="arcgis_ios_sdk_samples" customModuleProvider="target" sceneMemberID="viewController">
```

Comment:
Please add constraints to position the subviews of the cells.

![screen shot 2018-10-17 at 12 12 59 pm](https://user-images.githubusercontent.com/2257493/47110586-1d6b2d00-d206-11e8-9066-872440413084.png)